src/components/plots/linePlot.ts

```diff
@@ -95,6 +95,30 @@ export module Plot {
       return ["x", "y"];
     }
 
+    protected _getClosestPlotData(queryPoint: Point, datasetKeys: string[], withinValue = Infinity) {
```

Comment:
Note that the implementation here uses plotData.selection to mark the closestSelection.  This does not work in the abstractPlot case since there could be multiple selections in plotData.selection whereas in linePlot there is only the line.  That is why the abstractPlot uses the closestIndex idea.